ci/official/envs/release_linux_arm64_cpu_py310

```diff
@@ -0,0 +1,9 @@
+TFCI_BAZEL_COMMON_ARGS="--config release_arm64_linux --config tf_public_cache_push --config resultstore --repo_env=TF_PYTHON_VERSION=$TFCI_PYTHON_VERSION"
+TFCI_BAZEL_TARGET_SELECTING_CONFIG_PREFIX=linux_arm64
+TFCI_BUILD_PIP_PACKAGE_ARGS="--cpu --nightly_flag"
```

Comment:
some `--nightly_flag` i think should be removed from the release configs

Reply:
You're right, thanks. Resolved in latest version.